pkg/api/grpc/proxy/handler_test.go

```diff
@@ -74,7 +74,10 @@ func metricsCleanup() {
 		serviceInvocationResponseRecvName)
 }
 
-var testLogger = logger.NewLogger("proxy-test")
+var (
+	testLogger = logger.NewLogger("proxy-test")
+	_          = diag.SetLatencyDistribution(nil)
```

Comment:
Is there a strong reason this should be a global variable rather than done on a per test basis?
Global state in tests is very burdensome/error prone long term, especially since `diag` itself is globally stateful (which we should address eventually).

Reply:
don't think so, let me revisit